lms/djangoapps/learner_home/mock_data.json

```diff
@@ -16,7 +16,6 @@
             "enrollment": {
                 "accessExpirationDate": "11/11/3030",
```

Comment:
should we now have "hasAccess" in here as well?

Reply:
Good catch, will update